### PR TITLE
Wiki premium features

### PR DESCRIPTION
**Pull request by itIsJim**

### CODE REVIEW

File: content/en/seachat/seachat-manual/02-create-agent/03-advanced-settings/03-agent-memory.md [hunk 1]
```diff
@@ -12,13 +12,49 @@ aliases:
...
+    display: block;
+    unicode-bidi: isolate;
+    box-sizing: border-box;
+   font-size: .8rem;
+   margin-left: .5rem;
+        align-content: center;
+   ">
+   <div>Premium</div>
+</div>
+</div>
```
Comment (amy17519) [inline content/en/seachat/seachat-manual/02-create-agent/03-advanced-settings/03-agent-memory.md:48]: Thanks @itIsJim this is really cool. Can you please change the text to:

This feature is only available on <Premium> <Enterprise> plans.

File: content/en/seachat/seachat-manual/03-add-knowledge/02-spreadsheet-upload.md [hunk 2]
```diff
@@ -11,6 +11,10 @@ aliases:
   - /en/seachat/seachat-manual/03-add-knowledge/02-spreadsheet-upload
 ---
 
+> 🧭 **File Size Rule**
+> 
+> According to your subscription plan, you can upload files up to a certain size. If you exceed the file upload limitation, you will receive an error message. Please refer to the [File Size Rule](https://wiki.seasalt.ai/seachat/seachat-manual/03-add-knowledge/file-size-rule/) for more information.
```
Comment (amy17519) [inline content/en/seachat/seachat-manual/03-add-knowledge/02-spreadsheet-upload.md:16]: Your file size limit for each uploaded document varies depending on your subscription plan. If you exceed the file upload limitation, you will receive an error message. Please reduce the size of your file before uploading again. Please refer to the [File Size Rule](https://wiki.seasalt.ai/seachat/seachat-manual/03-add-knowledge/file-size-rule/) for more information.

Comment (amy17519) [reply content/en/seachat/seachat-manual/03-add-knowledge/02-spreadsheet-upload.md:16]: change to all section

File: content/en/seachat/seachat-manual/03-add-knowledge/file-size-rule.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+According to your subscription plan, you can upload files up to a certain size. If you exceed the file size limit, you will receive an error message.
+
+To understand the specific features in different plans, please refer to the [Pricing and Plans](https://wiki.seasalt.ai/seachat/seachat-payments/pricing-plans/).
+
+The file size rule for each plan are as follows:
+
+- **Free** - Knowledge base includes up to 10 documents (200KB max) and 200k tokens
+- **Standard** - Knowledge base includes up to 100 documents (1 MB max) and 1 million tokens
+- **Premium** - Knowledge base includes up to 500 documents (10 MB max) and 5 million tokens
```
Comment (amy17519) [inline content/en/seachat/seachat-manual/03-add-knowledge/file-size-rule.md:22]: can you add details on how to check the # of tokens of a document

File: content/en/seachat/seachat-payments/pricing-plans.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
+---
+title: "Pricing and Plans"
+description: ""
```
Comment (amy17519) [inline content/en/seachat/seachat-payments/pricing-plans.md:3]: add description: SeaChat pricing details

File: content/en/seachat/seachat-manual/03-add-knowledge/file-size-rule.md (hunk 3 above)
Comment (xuchen) [reply content/en/seachat/seachat-manual/03-add-knowledge/file-size-rule.md:22]: add conclusion from https://github.com/seasalt-ai/ngChat/issues/7862

basically:
for 4o **models**, each row is 8K tokens. for 3.5 model, each spreadsheet row is 2K tokens.

File: content/en/seasaltApi/seasalt-api/01-seachat-api-intro.md [hunk 5]
```diff
@@ -11,6 +11,40 @@ menu:
 weight: 103
 toc: true
 ---
+<div style="display: flex; align-self: flex-end; align-items: baseline">
```
Comment (xuchen) [inline content/en/seasaltApi/seasalt-api/01-seachat-api-intro.md:14]: this reminds me: can you add a section under "Resources" called "SeaChat APIs"? Then point to this page
image

**Review by xuchen — APPROVED** [review]

**Review by amy17519 — APPROVED** [review]

**Pull request merged**